src/Arcus.Security.Providers.AzureKeyVault/KeyVaultSecretProvider.cs

```diff
@@ -144,8 +178,14 @@ protected async Task<IKeyVaultClient> GetClientAsync()
                     _keyVaultClient = await _authentication.AuthenticateAsync();
                 }
 
+                Logger.LogInformation("Authenticated with the Azure Key Vault");
                 return _keyVaultClient;
             }
+            catch (Exception exception)
```

Comment:
Don't we log this in the authenticate itself so that feels enough maybe?

Reply:
The authentication is an interface, so can be whatever code they inject in it, right.